standalone/README.md

```diff
@@ -0,0 +1,54 @@
+[![PILZ](../img/pilz-logo.png)](https://www.pilz.com)
+
+# PILZ safety laser scanner PSENscan Standalone C++ Library
+
+The PSENscan Standalone C++ Library allows you to easily integrate the PSENscan safety laser scanner into your application without depending on ROS. You will be able to read both distance and intensity values from the PSENscan safety laser scanner, which can subsequently be used for navigation algorithms and much more.
+
+## Table of Contents
+
+1. [Get Started](#get-started)
+2. [Usage example](#usage-example)
+3. [C++ API](#c++-api)
+
+## Get Started
+First, clone this repository:
+```
+git clone https://github.com/PilzDE/psen_scan_v2.git
+```
+
+Next, navigate to the standalone folder and create a `build` directory.
+```
+cd psen_scan_v2/standalone/
+mkdir build
+```
+
+Lastly, execute the following:
+```
+cd build/ && cmake .. && make
+```
+
+The library should now be built and available under `psen_scan_v2/standalone/build/libpsen_scan_v2_standalone.a`
+
+## Usage example
+To build the example application, which prints distance data to the screen, you can follow the first two steps as in [Get Started](#get-started) if you haven't done so already. The last command will be different:
+```
+cd build/ && cmake -DBUILD_EXAMPLE=ON .. && make
```

Comment:
Why add an extra flag to build the example? I think we should always build that, to keep getting started as easy as possible...

Reply:
In that case it would mean that the standalone example program will also be built when building the ros part. Would this be intended?

Reply:
It doesn't harm imho. Do you see any disadvantages?

Reply:
A disadvantage could be when typing `rosrun psen_scan_v2` in the console and double-tapping `TAB` for autocompletion.
When running it with always building the example app:
```
psen_scan_v2_app   psen_scan_v2_node
```
1. User might be confused at the difference.
2. psen_scan_v2_app is not actually a ROS application. User might try to run it expecting some kind of ROS publication.

Reply:
Depending on use case (apt pkg vs git checkout) this might not be an issue since the target is not installed.

Reply:
or build it on default and have a flag to NOT build it for catkin builds

Reply:
> or build it on default and have a flag to NOT build it for catkin builds

Done.